src/main/java/com/conveyal/datatools/manager/jobs/AutoDeployJob.java

```diff
@@ -95,18 +92,10 @@ public void jobLogic() {
                     );
                     continue;
                 }
-                List<String> feedVersionIdsForDeployment = getFeedVersionsForDeployment(project);
-                if (feedVersionIdsForDeployment.isEmpty()) {
-                    LOG.info("Project {} skipped for auto deployment as no newer feed versions are available!",
-                        project.name);
-                    continue;
-                }
                 // Send deployment (with new feed version) to most recently used server.
                 String latestServerId = deployment.latest().serverId;
                 OtpServer server = Persistence.servers.getById(latestServerId);
                 if (server != null) {
-                    deployment.feedVersionIds = new ArrayList<>();
-                    deployment.feedVersionIds.addAll(feedVersionIdsForDeployment);
```

Comment:
This commit here got rid of the process of updating the deployment to have all of the updated feed versions. It wasn't quite correct, but the current state is now back at a point where a deployment's feed versions don't get updated. There needs to be some kind of logic to update all the feed versions of a deployment to the latest feed version available for a particular feed source associated with the deployment's current feed versions. Also, the updated deployment should be persisted before queueing the deploy job.

Reply:
Issue addressed in Evan's PR.